modules/menu/dynamic_menu.c

```diff
@@ -15,19 +15,19 @@ static int set_audio_bitrate(struct re_printf *pf, void *arg)
 	struct ua *ua = carg->data ? carg->data : menu_uacur();
 	struct call *call;
 	uint32_t bitrate = str_isset(carg->prm) ? atoi(carg->prm) : 0;
-	int err;
 
 	call = ua_call(ua);
 	if (call) {
-		err = re_hprintf(pf, "setting audio bitrate: %u bps\n",
+		(void)re_hprintf(pf, "setting audio bitrate: %u bps\n",
 				 bitrate);
 		audio_set_bitrate(call_audio(call), bitrate);
 	}
 	else {
-		err = re_hprintf(pf, "call not found\n");
+		(void)re_hprintf(pf, "call not found\n");
+		return EINVAL;
 	}
 
-	return err;
+	return 0;
```

Comment:
can you please make a separate PR for this block ?

it is not related to the main topic of this PR.